src/TestOverloading1.java

```diff
@@ -0,0 +1,36 @@
+import java.util.Scanner;
+
+public class TestOverloading1 {
+
+    static void testParameters(float a, float b, float c) {
```

Comment:
Incorrect method implementation

Reply:
static void method(int input) {
    System.out.println("The sum of each number for " + input + " is: " +
        ((input % 10) + ((input / 10) % 10) + ((input / 100) % 10)));
  }